adele-ticket-master-api/src/main/java/com/microservicesteam/adele/ticketmaster/events/TicketsEvent.java

```diff
@@ -2,9 +2,20 @@
 
 import java.util.List;
 
+import com.fasterxml.jackson.annotation.JsonSubTypes;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
 import com.microservicesteam.adele.messaging.events.Event;
 import com.microservicesteam.adele.ticketmaster.model.Position;
 
-interface TicketsEvent extends Event {
+@JsonTypeInfo(use=JsonTypeInfo.Id.NAME,
+        include=JsonTypeInfo.As.PROPERTY,
+        property="type")
```

Comment:
@kelemena have you tried to use `@class` here? I could see some examples where it was used and so `@JsonTypeName` wasn't necessary on the child classes.
http://www.studytrails.com/java/json/java-jackson-serialization-polymorphism/